mobile/native/src/ln_dlc/channel_status.rs

```diff
@@ -19,19 +18,14 @@ pub enum ChannelStatus {
     /// there was a previous channel nor does it imply that a previous channel was completely
     /// closed i.e. there might be pending transactions.
     NotOpen,
-    /// There is a vanilla Lightning channel, without a subchannel.1
-    ///
-    /// This means that it is possible to use the Lightning channel for payments and that the
-    /// channel can be upgraded to an LN-DLC channel.
-    LnOpen,
-    /// There is an LN-DLC channel.
-    ///
-    /// This corresponds to an app which currently has an open position as a result of trading.
-    LnDlcOpen,
-    /// The LN-DLC channel is in the process of being force-closed.
-    LnDlcForceClosing,
-    /// The LN-DLC channel is in an inconsistent state.
-    Inconsistent,
+    /// DLC Channel is open
+    Open,
+    /// DLC Channel is open and has an open position
+    WithPosition,
+    /// DLC Channel is currently in progress of being renewed
+    Pending,
```

Comment:
According to the logic below, I think the channel can be `Pending` if it's being renewed or if it is being settled.

"Pending" makes me think of a channel that is going from `NotOpen` to `Open`, so maybe we should rename this to `BeingUpdated` or something like that.

Reply:
Or we could even introduce `Renewing` and `Settling`.

I think we may even want to use the same model that `rust-dlc` uses now that we are not dealing with both LN channels and DLC channels.

Reply:
> Or we could even introduce Renewing and Settling.

Yes, I thought we'd introduce these temporary state with https://github.com/get10101/10101/issues/1791

> I think we may even want to use the same model that rust-dlc uses now that we are not dealing with both LN channels and DLC channels.

That makes sense 🙈